setup.py

```diff
@@ -28,6 +28,46 @@ def get_setup_version(reponame):
         print("WARNING: param>=1.6.0 unavailable. If you are installing a package, this warning can safely be ignored. If you are creating a package or otherwise operating in a git repository, you should install param>=1.6.0.")
         return json.load(open(version_file_path, 'r'))['version_string']
 
+def build_custom_models():
```

Comment:
I moved this up in the file to make the setup.py easier to read.

Reply:
Ok

Reply:
Merge at your leisure!